src/components/LogsComponent.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  
+  <div class="items-start justify-start">
+    <div class="q-pa-md">
+      <q-table
+       v-model:selected="selected"
+        :title="$t('logsPage.title')"
+        :rows="row"
+        :columns="column"
+        row-key="name"
+        selection="multiple"
+        hide-header
+        hide-bottom
+      >
+        <template v-slot:top>
+          <div class=" text-h5">
+            {{ $t('logsPage.title') }}
+          </div>
+          
+        </template>
+      </q-table>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref } from 'vue'
+
+const selected = ref([]);
+
+const column = [
+  {
+    name: 'name',
+    required: true,
+    label: 'Name_Log',
+    align: 'left',
+    field: (row: { name: any; }) => row.name,
+    format: (val: any) => `${val}`,
```

Comment:
daj po prostu val jako zwrotke, a jesli chcesz typowo stringa to zmien moze typ i po prostu swroc val.toString()